Cargo.toml

```diff
@@ -7,15 +7,16 @@ license = "Apache-2.0"
 repository = "https://github.com/oxidecomputer/diesel-dtrace.git"
 
 [dependencies]
-diesel = { version = "2.1.0", features = [ "r2d2", "i-implement-a-third-party-backend-and-opt-into-breaking-changes" ] }
+diesel = { version = "=2.1.6", features = [ "r2d2", "i-implement-a-third-party-backend-and-opt-into-breaking-changes" ] }
```

Comment:
why is this necessary?

Reply:
@ahl the issue is that async-bb8-diesel off `main` will make Cargo.lock rely on diesel 2.2, which will break this building this dep b/c the impl on `Connection` (https://github.com/oxidecomputer/diesel-dtrace/blob/main/src/lib.rs#L117) has new functions in 2.2. 

I attempted to just set async-bb8 to the rev on omicron, but without the `=2.1.6`, I was still having it float to 2.2 deps.